src/Machine.Specifications.Runner.ReSharper/Machine.Specifications.Runner.ReSharper.csproj

```diff
@@ -17,13 +17,13 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <SdkVersion>2019.3.0</SdkVersion>
+    <SdkVersion>2020.1.0</SdkVersion>
     <WaveVersionBase>$(SdkVersion.Substring(2,2))$(SdkVersion.Substring(5,1))</WaveVersionBase>
     <WaveVersion>$(WaveVersionBase).0.0</WaveVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="JetBrains.ReSharper.SDK" Version="2019.3.4" PrivateAssets="All" />
+    <PackageReference Include="JetBrains.ReSharper.SDK" Version="2020.1.0-eap08" PrivateAssets="All" />
```

Comment:
They've officially released Rider 2020.1, so should not be eap enymore.

Reply:
Thanks, it's been 4 hours since the release. Give me a break.